src/java/org/apache/cassandra/tools/nodetool/GuardrailsConfigCommand.java

```diff
@@ -59,51 +61,107 @@ public static class GetGuardrailsConfig extends GuardrailsConfigCommand
         allowedValues = { "values", "thresholds", "flags", "others" })
         private String guardrailCategory;
 
-        @Arguments(description = "Specific names or guardrails to get configuration of.")
+        @Option(name = { "--verbose", "-v" })
```

Comment:
Now I'm thinking that `--verbose` is probably unclear. Maybe we could choose a better name?

`--expand-thresholds`
`--detailed`
`--show-threshold-details`

Reply:
--detailed is fine

Reply:
changed to --expand.